tests/ui/test_home.py

```diff
@@ -21,3 +19,52 @@ def test_most_popular_extensions_are_sorted_by_users(
     extensions = page.most_popular.extensions
     sorted_by_users = sorted(extensions, key=lambda e: e.users, reverse=True)
     assert sorted_by_users == extensions
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_that_clicking_on_addon_name_loads_details_page(
+        my_base_url, selenium, initial_data):
+    page = Home(selenium, my_base_url).open()
+    page.click_on_first_addon()
+    page.wait_for_page
+    assert 'addon' in selenium.current_url
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_that_featured_themes_exist_on_the_home(
+        my_base_url, selenium, generate_themes):
+    page = Home(selenium, my_base_url).open()
+    assert u'Featured Themes See all \xbb', 'Featured Themes region title \
+        doesn\'t match' == page.featured_themes_title
+    assert page.featured_themes.count >= 6
```

Comment:
So ideally this would be `assert len(page.featured_themes.themes) >= 6`. I also wonder if this check is correct, as 7 themes would look odd and unbalanced on the home page.. perhaps it should be exactly 6?